### PR TITLE
Make the Ballerina Project Tooling Friendly

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
$Subject

Related Issue: [#1746](https://github.com/ballerina-platform/ballerina-standard-library/issues/1746)

## Checklist
- [x] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 [package]
 org = "ballerinax"
 name = "nats"
-version = "@toml.version@"
+version = "2.0.0"
```
Comment (aashikam) [inline ballerina/Ballerina.toml:4]: Committed intentionally?

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 distribution = "slbeta3"
 
 [[platform.java11.dependency]]
-path = "./lib/jnats-@nats.client.version@.jar"
+path = "./lib/jnats-2.8.0.jar"
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/nats-native-@project.version@.jar"
+path = "../native/build/libs/nats-native-2.0.0-SNAPSHOT.jar"
```
Comment (aashikam) [inline ballerina/Ballerina.toml:11]: This whole file actually

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (ThisaruGuruge) [reply ballerina/Ballerina.toml:4]: Yes, that's the purpose of the PR.

Comment (aashikam) [reply ballerina/Ballerina.toml:4]: yeah figured 😂

**Review by aashikam — APPROVED** [review]

**Pull request merged**